example_configs/text2text/en-de/en-de-convs2s.py

```diff
@@ -0,0 +1,197 @@
+from __future__ import absolute_import, division, print_function
```

Comment:
Can you add "from \_\_future\_\_ import unicode_literals" to all files? And check your python2 fix again, because it might affect that code.